source/configuration/modules/imjournal.rst

```diff
@@ -245,7 +245,7 @@ WorkAroundJournalBug
    :widths: auto
    :class: parameter-table
 
-   "binary", "off", "no", "none"
```

Comment:
is this a doc fix or a change-of-behavior?

Reply:
Doc fix - the actual default was changed in https://github.com/rsyslog/rsyslog/pull/3566

Reply:
thx!